File: java/src/main/java/vsdk/toolkit/io/context/ParseOptionsContext.java
package vsdk.toolkit.io.context;

import vsdk.toolkit.scene.RadianceMethod;

public class ParseOptionsContext {
    public RadianceMethod radianceMethod;
    public boolean singleSided;
    public int numberOfQuarterCircleDivisions;
    public boolean monochrome;

    public ParseOptionsContext() {
        radianceMethod = null;
        singleSided = false;
        numberOfQuarterCircleDivisions = 0;
        monochrome = false;
    }
}

File: typescript/src/main/vsdk/toolkit/io/context/ParseOptionsContext.ts
import { RadianceMethod } from "../../scene/RadianceMethod";

export class ParseOptionsContext {
  public radianceMethod: RadianceMethod | null;
  public singleSided: boolean;
  public numberOfQuarterCircleDivisions: number;
  public monochrome: boolean;

  public constructor() {
    this.radianceMethod = null;
    this.singleSided = false;
    this.numberOfQuarterCircleDivisions = 0;
    this.monochrome = false;
  }
}
